Exercises/JonasBecker/exercises_01.py

```diff
@@ -0,0 +1,111 @@
+# use this template for completing todays exercises
+# make a copy of the file, in a folder with your own name
+# add code below to answer the questions; for each question there should be
+# a single print() which will give the answer(s) to that question.
+
+# 1)
+# what are the quotient and remainder after integer division of 2711274328912 by 23369245575
+print(
+    f"1)\n"
+    f"Result of integer division of 2711274328912 by 23369245575\n"
+    f"quotient: {2711274328912 / 23369245575},\n"
+    f"quotient_floor: {2711274328912 // 23369245575},\n"
+    f"remainder: {2711274328912 % 23369245575}"
+)
+
+# 2)
+# assume the variable below is a temperature in degrees Fahrenheit. Print the same temperature converted to °C.
+t_fahrenheit = 451
+print(
+    f"\n2)\n{t_fahrenheit} degree fahrenheit equals to ~{(t_fahrenheit-32)*(5/9):.2f} degree Celsius"
```

Comment:
very nice that you used a format string with digits specifier. Now being very nit-picky: The number of significant digits in the output should however match the number of significant digits in the input (assuming the conversion constants are exact, which they are).

Also: In general these magic numbers (`32`, `5/9`) should be defined as variables, typically with an all-caps name to indicate they are constant, and the name should explain what they are